buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/DistributedPerformanceTest.groovy

```diff
@@ -139,12 +133,29 @@ class DistributedPerformanceTest extends PerformanceTest {
         }
     }
 
+    private File generateResultJson() {
+        File resultJson = new File(workerTestResultsTempDir, 'results.json')
+        List<Map> resultData = finishedBuilds.collect { workerBuildId, xmlroot ->
+            // org.gradle.performance.results.ScenarioBuildResultData
+            [
+                scenarioName: findScenarioNameInXml(xmlroot),
```

Comment:
Do we really need to parse the scenario name from the returned XML? We know that when we trigger the build, right?